### ISSUE TITLE
Post Paramters

### ISSUE THREAD

**Issue by jbolles5** [michelson/Dante#76]
I have a tree going here: https://github.com/RundownApp/Dante/tree/add_save_method

I was able to successfully add an option to allow you to specify the name of the parameter that gets posted on save, but would like to be able to specify other parameters as well. My lack of a background in CoffeeScript is also not helping my cause.

Another option might be to specify an auto save method client side so you can construct your own AJAX. That was my original direction, but there is already save_method for specifying POST/GET/etc and I was not creative enough to come up with a different option name.

**Comment by michelson** [comment]
Hi @jbolles5 , check this options for auto save behavior:

store_url: default: to none , url to store data with interval
store_method default: to POST , http verb to use when store_url is present.
store_interval: default: 15000 (15 secs), used when store_url is present.

can you explain what would be the use case in which you need to send other params ?

**Comment by jbolles5** [comment]
Sure. I am working on an implementation where a user would be assigned a task to write a blog post. They would use Dante to write the post, however we would need to pass which post they were working on, along with the body of the post, back to the server so that the new version can be created properly. The current implementation only posts the body of the post back to the server without any context of what it is for.

**Comment by michelson** [comment]
got it , 

maybe you can use a form , and when you submit it add the editor content to the hidden field in order to save the post.

example: 
```javascript
//on form submit
form = $("#form");
form.find("#post_excerpt").val( editor.getContent() );
```

**Comment by walterdavis** [comment]
Wouldn't that be up to the URL you post to? What is your back-end? Rails works like this (assuming a pages_controller and regular REST URLs):

POST /pages (new page created, redirect to pages/[new id])
PUT or PATCH /pages/42 (update the existing page, redirect to pages/42)
DELETE /pages/42 (removes the page, redirect to pages/)

Walter

**Comment by jbolles5** [comment]
Have thought about both solutions. I am using rails, but the object I am creating is a version of the blog post, so unfortunately there is some extra meta that has to get passed in. I am thinking of using query strings on the URL.

**Comment by michelson** [comment]
@jbolles5 the best way is to use a form, before submit event copy editor content to some hidden text field along with other fields

**Comment by michelson** [comment]
fixed in #69
